scripts/edit-datapackage-json.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+from pprint import pprint
+
+import seedcase_sprout.core as sp
+
+properties = sp.PackageProperties(
```

Comment:
It probably doesn't make sense to copy the exact same properties from above and update the existing `datapackage.json` file with the same content. Maybe intentionally introduce a mistake in the creation step and fix it down here?

Reply:
I obviously committed the version where I changed it back, and not the one where I had made changes. They were there in a previous save. I'll put them back.